specification/cognitiveservices/data-plane/ContentModerator/v1.0/ContentModerator.json

```diff
@@ -1816,42 +2286,286 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "timescale",
+            "description": "Timescale of the video .",
+            "in": "query",
+            "required": false,
+            "type": "number"
+          },
+
+          {
+            "name": "frameImageZip",
+            "description": "Zip file containing frame images.",
+            "in": "formData",
+            "required": true,
+            "type": "file"
+
           },
           {
-            "$ref": "#/parameters/contentTypeHeader"
+            "name": "frameMetadata",
+            "description": "Metadata of the frame.",
+            "in": "formData",
+            "required": true,
+            "type": "string"
+
           },
           {
-            "$ref": "#/parameters/ImageStreamParameter"
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
           }
         ],
         "consumes": [
-          "image/gif",
-          "image/jpeg",
-          "image/png",
-          "image/bmp",
-          "image/tiff"
+          "multipart/form-data"
         ],
         "responses": {
-          "200": {
-            "description": "OK",
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response.",
             "schema": {
-              "$ref": "#/definitions/AddImage"
+              "$ref": "#/definitions/APIError"
             }
           }
         },
-        "x-ms-examples": {
-          "Add Image request": {
-            "$ref": "./examples/AddImageStreamResource.JSON"
-          }
-        },
+       
         "produces": [
           "application/json"
         ]
+
       }
     }
+    
   },
   "definitions": {
+    "Frames": {
+      "description": "The response for a Get Frames request.",
+      "properties": {
+        "reviewId": {
+          "description": "Id of the review.",
+          "type": "string"
+        },
+        "videoFrames": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Frame"
+          }
+        }
+      }
+    },
+    "Frame": {
+      "description": "Video frame property details.",
+      "properties": {
+        "timestamp": {
+          "description": "Timestamp of the frame.",
+          "type": "string"
+        },
+        "frameImage": {
+          "description": "Frame image.",
+          "type": "string"
+        },
+        "metadata": {
+          "description": "Array of KeyValue.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KeyValuePair"
+          }
+        },
+        "reviewerResultTags": {
+          "description": "Reviewer result tags.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Tag"
+          }
+        },
+        "frameImageBytes": {
+          "description": "Frame Image Bytes.",
+          "type": "string"
```

Comment:
Should this be `byte`?

Reply:
Just to explain a little - not every byte can be represented as an ASCII string in the context of an image - converting a byte array to string often involves data loss: https://stackoverflow.com/a/9098905 unless it's base 64 encoded.

Reply:
I have removed this parameter since we are going to remove it from our service.(No longer used)